commerce-account-web/src/main/resources/META-INF/resources/view_account_list.jsp

```diff
@@ -66,6 +67,28 @@ request.setAttribute("view.jsp-filterPerAccount", false);
 			}
 		);
 
+		Liferay.provide(
+			window,
+			'toggleActiveCommerceAccount',
+			function(id) {
+				document.querySelector('#<portlet:namespace /><%= Constants.CMD %>').value = 'setActive';
+				document.querySelector('#<portlet:namespace />commerceAccountId').value = id;
+
+				submitForm(document.<portlet:namespace />commerceAccountFm);
+			}
+		);
+
+		Liferay.provide(
+			window,
+			'setCurrentAccount',
+			function(id) {
+				document.querySelector('#<portlet:namespace /><%= Constants.CMD %>').value = 'setCurrentAccount';
+				document.querySelector('#<portlet:namespace />commerceAccountId').value = id;
+
+				submitForm(document.<portlet:namespace />commerceAccountFm);
+			}
+		);
```

Comment:
alphabetize by function name (i.e. `toggleActiveCommerceAccount` and `setCurrentAccount`).